### PR TITLE
Bug stomp 6 21

### PR DESCRIPTION
**Pull request by jwoolbright23**

### CODE REVIEW

**Comment by jwoolbright23** [comment]
Addressed the following issues in this PR:
1. #89 
2. #118 
3. #92 #93 
4. #97 
5. #101 #102

File: content/assignments/candidate-testing/task-1/index.md [hunk 1]
```diff
@@ -70,11 +70,21 @@ To see which tests you passed, scroll up in your terminal.  You will see details
    <img src="../images/Part1-tests-list.png" alt="Results of tests with only task 1 completed" width=50% />
 {{< /rawhtml >}}
 
+[]()
```
Comment (gildedgardenia) [inline content/assignments/candidate-testing/task-1/index.md:73]: What is this here for? It is creating extra white space.

File: content/assignments/candidate-testing/task-2/index.md [hunk 2]
```diff
@@ -35,6 +35,12 @@ Now that your small app is working, expand it to deal with multiple questions. T
    {{% notice blue "Note" "rocket" %}} 
    Make sure your loops work properly before moving on to task 3.
 
+   Run the following command to make sure the app is running without errors, if you do encounter any errors:
```
Comment (gildedgardenia) [inline content/assignments/candidate-testing/task-2/index.md:38]:
```suggestion
   Run the following command to make sure the app is running without errors:
```

File: content/fetch/exercises/_index.md [hunk 3]
```diff
@@ -103,6 +103,10 @@ Add this preliminary HTML to your `fetch_planets` document:
    ```
    {{% /expand %}}
 
+{{% notice blue Note "rocket" %}}
+You should not expect a response, take note of the error that you receive from the console!
```
Comment (gildedgardenia) [inline content/fetch/exercises/_index.md:107]:
```suggestion
You should not expect a response. Take note of the error that you see in the console!
```

**Review by gildedgardenia — APPROVED** [review]
These changes look good! I did have one question for you, but approving your work for now.

File: content/assignments/candidate-testing/task-1/index.md (hunk 1 above)
Comment (jwoolbright23) [reply content/assignments/candidate-testing/task-1/index.md]: fixed

**Pull request merged**